core/equipment.c

```diff
@@ -229,62 +246,74 @@ int find_best_gasmix_match(struct gasmix mix, const struct cylinder_table *cylin
  * we should pick up any other names from the dive
  * logs directly.
  */
-struct tank_info_t tank_info[MAX_TANK_INFO] = {
-	/* Need an empty entry for the no-cylinder case */
-	{ "", },
-
+static void add_default_tank_infos(struct tank_info_table *table)
+{
 	/* Size-only metric cylinders */
-	{ "10.0ℓ", .ml = 10000 },
-	{ "11.1ℓ", .ml = 11100 },
+	add_tank_info_metric(table, "10.0ℓ", 10000, 0);
+	add_tank_info_metric(table, "11.1ℓ", 11100, 0);
 
 	/* Most common AL cylinders */
-	{ "AL40", .cuft = 40, .psi = 3000 },
-	{ "AL50", .cuft = 50, .psi = 3000 },
-	{ "AL63", .cuft = 63, .psi = 3000 },
-	{ "AL72", .cuft = 72, .psi = 3000 },
-	{ "AL80", .cuft = 80, .psi = 3000 },
-	{ "AL100", .cuft = 100, .psi = 3300 },
+	add_tank_info_metric(table, "AL40", 40, 3000);
+	add_tank_info_metric(table, "AL50", 50, 3000);
+	add_tank_info_metric(table, "AL63", 63, 3000);
+	add_tank_info_metric(table, "AL72", 72, 3000);
+	add_tank_info_metric(table, "AL80", 80, 3000);
```

Comment:
Shouldn't these use add_tank_info_imperial?

Reply:
Yes, of course. Thanks for pointing that out. Fix in #3105.